scripts/setup-hooks.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+ln -sf ../../.hooks/pre-commit.sh .git/hooks/pre-commit
```

Comment:
Is this path right? From where is this run?

Reply:
https://github.com/nvim-tree/nvim-web-devicons/pull/218/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R65

However... I think there is a better method https://github.com/nvim-tree/nvim-web-devicons/pull/218#discussion_r1126748422